lib/ProductOpener/Display.pm

```diff
@@ -1325,9 +1336,25 @@ sub display_error($error_message, $status) {
 		status => $status,
 		page_type => "error",
 	});
+	return;
+}
+
+
+=head2 display_error_and_exit ( $error_message, $status )
+
+Display an error message using the site template, and terminate the request immediately.
+
+Any code after the call to display_error_and_exit() will not be executed.
+
+=cut
+
+sub display_error_and_exit($error_message, $status) {
+
+	display_error($error_message, $status);
 	exit();
```

Comment:
Maybe we could set whatever status if we do `exit(Apache2::Const::OK)` ? (or is it the default already ?)

See https://perl.apache.org/docs/2.0/api/ModPerl/Util.html#C_exit_

Reply:
sorry, it seems return code is ignored :-D